src/utils/exportSQL/postgres.js

```diff
@@ -32,6 +32,8 @@ export function toPostgres(diagram) {
               `${exportFieldComment(field.comment)}\t"${
                 field.name
               }" ${field.type}${field.isArray ? " ARRAY" : ""}${field.notNull ? " NOT NULL" : ""}${field.unique ? " UNIQUE" : ""}${
+                field.increment && field.primary ? " GENERATED BY DEFAULT AS IDENTITY" : ""
```

Comment:
No need for `&& field.primary`. It doesn't have to be a primary key to be auto incremented